apps/ui/src/components/PageLayout/index.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { ValidLanguage } from '@/i18n';
+import Container from '@/components/Container';
+import Header from '@/components/Header';
+import Footer from '@/components/Footer';
+import styles from './PageLayout.module.scss';
+
+const PageLayout = async ({
+  children,
+  lang,
+}: {
+  children: React.ReactNode;
+  lang: ValidLanguage;
+}) => {
+  return (
+    <>
+      <Container>
+        <Header lang={lang} />
+        <main className={styles.main}>{children}</main>
+        <Footer lang={lang} />
+      </Container>
+    </>
```

Comment:
since all those components are withing the one `Container`, we don't need a `Fragment` here.
(this suggestion is for illustration only, as it doesn't re-indent things)
```suggestion
      <Container>
        <Header lang={lang} />
        <main className={styles.main}>{children}</main>
        <Footer lang={lang} />
      </Container>
```